config/install/fvm.settings.yml

```diff
@@ -38,3 +41,27 @@ block_content__cards: 1
 block_content__cards__view_modes:
   cards_grid_3xn: cards_grid_3xn
   preview: preview
+media__video: 0
+media__video__view_modes:
+  gallery: 0
+paragraph__form: 0
+paragraph__form__view_modes:
+  preview: 0
+paragraph__gallery: 1
+paragraph__gallery__view_modes:
+  gallery: gallery
+  gallery_carousel: gallery_carousel
+  preview: 0
+paragraph__gallery_carousel: 0
+paragraph__gallery_carousel__view_modes:
+  preview: 0
+paragraph__view: 0
+paragraph__view__view_modes:
+  preview: 0
+node__gallery: 0
+node__gallery__view_modes:
+  teaser: 0
+paragraph__cards_gallery: 1
+paragraph__cards_gallery__view_modes:
+  gallery: gallery
+  gallery_carousel: gallery_carousel
```

Comment:
The disabled items shall not be part this config. As they may not be enabled on new site install.